pacini: add page (#18449)
tldr-pages/tldr: pages/linux/pacini.md

# pacini

> > Query tool for pacman-style configuration files like /etc/pacman.conf.

Reviewer: ```suggestion
> Query pacman-style configuration files.
```
The original was better. See the style guide https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#imperative-mood

Author: okkay sir

> More information: <https://github.com/andrewgregory/pacutils/blob/master/doc/pacini.pod>.

> - Show the full parsed configuration file (default: /etc/pacman.conf):

Author: let me add another command

# pacini

> Query pacman-style configuration files.
> More information: <https://github.com/andrewgregory/pacutils/blob/master/doc/pacini.pod>.

- Show the full parsed configuration file (default: `/etc/pacman.conf`):

`pacini`

- List configured sections:

`pacini --section-list`

- Query a custom configuration file:

> `pacini /path/to/config.conf`

Author: it's rooted at the file system root directory

# pacini

> Query pacman-style configuration files.
> More information: <https://github.com/andrewgregory/pacutils/blob/master/doc/pacini.pod>.

- Show the full parsed configuration file (default: `stdin`):

`pacini {{path/to/file}}`

- List configured sections:

`pacini --section-list`

> - Display directive names:
> 
> `pacini --verbose`

Reviewer 2: https://github.com/andrewgregory/pacutils/blob/master/doc/pacini.pod
`--verbose` does nothing without specifying a directive.
```suggestion
- Always show directive names even if only one directive is provided:

`pacini --verbose {{directive}}`
```

# pacini

> Query pacman-style configuration files.
> More information: <https://github.com/andrewgregory/pacutils/blob/master/doc/pacini.pod>.

> - Show the full parsed configuration file (default: `/etc/pacman.conf`):
> 
> `pacini`

Reviewer 2: Where did you get that default file from?
https://github.com/andrewgregory/pacutils/blob/master/doc/pacini.pod
> If file is omitted or `-` the configuration will be read from stdin.
```suggestion
- Show the full parsed configuration file (default: `stdin`):

`pacini {{path/to/file}}`
```

Author: i forgot sir where i got that, okkay let just update it with your request

- List configured sections:

`pacini --section-list`

- Display directive names:

`pacini --verbose`

- Display directives listed in section `name`:

`pacini --section=name`

- Query a custom configuration file:

> `pacini {{/path/to/config.conf}}`

Author: yes sir working under my system too
<img width="607" height="259" alt="image" src="https://github.com/user-attachments/assets/5c6ba5fd-b461-4570-beb5-efd988ab0201" />

i will update the code

> 
> - Query a custom configuration file:
> 
> `pacini {{path/to/config.conf}}`

Reviewer 2: Moved to the first example with my other suggestion
```suggestion
```

Author: okkay sir
